source/src/protocols/recon_design/recon_util.cc

```diff
@@ -45,31 +45,40 @@ get_designable_residues( core::pose::Pose & pose, std::string resfile ) {
 
 /// Based on a pose and the indices of all designable residues, get a
 /// string of all designable AAs concatenated
-std::string
+utility::vector1< std::string >
 get_designable_sequence ( core::pose::Pose & pose, utility::vector1< core::Size > designable_residues ) {
-	std::string sequence = "";
+	//std::string sequence = "";
+    utility::vector1< std::string > sequence;
 	for ( core::Size seqpos: designable_residues ) {
-		sequence += pose.residue( seqpos ).name1();
+		//sequence += pose.residue( seqpos ).name1();
+        sequence.push_back(pose.residue_type( seqpos ).base_name());
 	}
 	return sequence;
 }
 
 /// Based on a list of sequences from poses, get all the AAs present at
 /// the position given by position_no
 utility::vector1< std::string >
-get_candidate_AAs( utility::vector1< std::string > other_pose_sequences,
+get_candidate_AAs( utility::vector1< utility::vector1< std::string > > other_pose_sequences,
```

Comment:
You probably want to pass unchanged vectors by `const &` rather than by value -- it saves data copies. (It's a pre-existing issue here, but is exasperated by adding the nesting and larger strings.)